lib/form/paragraph3.flow

```diff
@@ -45,6 +45,20 @@ possible, and because rendering words in the same sprite improves the interword
 */
 
 
+// How extra space of paragraph line will be spread:
+// start — added at start of first word;
+// justify — equally distributed between words or added at the end(!) of the only word or last line;
+// end=1-start-justify — what's left will be at end of the line.
+// We might store start and end, but even this variant seems symmetrical,
+// the used one is more useful for calculations.
+GeneralAlignment(
+	start : double,
+	justify : double
```

Comment:
justify seems to be pure bool, not double.

Reply:
I have to agree.